certified-connectors/Redque/apiProperties.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions"
+    ],
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "<<Enter your client ID>>",
+          "scopes": [
+            "v1.public",
+            "offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://demoidentity.redque.com/connect/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://demoidentity.redque.com/connect/token"
+            },
+            "refreshUrl": {
+              "value": "https://demoidentity.redque.com/connect/token"
+            }
+          },
+          "properties": {
+            "IsFirstParty": "False"
+          }
+        },
+        "uiDefinition": {
+          "displayName": "Log in with RedQue Account",
+          "description": "Log in with RedQue Account",
+          "tooltip": "Log in with RedQue Account",
+          "constraints": {
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0db9d6",
```

Comment:
@jirisodomka this background color does not match with your icon. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector)